src/main/scala/is/hail/expr/Parser.scala

```diff
@@ -147,6 +147,39 @@ object Parser extends JavaTokenParsers {
     })
   }
 
+  def parseAnnotationExprsToAST(code: String, ec: EvalContext, expectedHead: Option[String]): (
+    Array[List[String]], Array[AST]) = {
+
+    val parsed = named_exprs(annotationIdentifier).parse(code)
+
+    val names = new Array[List[String]](parsed.size)
+    val asts = new Array[AST](parsed.size)
+
+    var i = 0
+    parsed.foreach { case (name, ast, _) =>
```

Comment:
You need to handle splat here.  See the discussion on splat here for example: https://hail.is/docs/stable/hail.VariantDataset.html#hail.VariantDataset.export_variants.

Reply:
can't splat go away in hail2?